subsys/bluetooth/host/gatt.c

```diff
@@ -1195,17 +1195,21 @@ ssize_t bt_gatt_attr_write_ccc(struct bt_conn *conn,
 {
 	struct _bt_gatt_ccc *ccc = attr->user_data;
 	struct bt_gatt_ccc_cfg *cfg;
-	u16_t value;
+	u16_t value = 0;
```

Comment:
Can remove the initialization. The value is always assigned.

Reply:
Fixed.